.cbmc-batch/include/openssl/ec.h

```diff
@@ -64,4 +64,7 @@ void ECDSA_SIG_free(ECDSA_SIG *sig);
 ECDSA_SIG *d2i_ECDSA_SIG(ECDSA_SIG **sig, const unsigned char **pp, long len);
 int i2d_ECDSA_SIG(const ECDSA_SIG *sig, unsigned char **pp);
 
+void initialize_max_encryption_size();
```

Comment:
Is this an openssl function, or a helper function you added?

Reply:
It's a helper function. It actually already existed in ec_override.c (and is analogous to the initialize_max_derivation_size() and initialize_max_signature_size() functions there) but I added it to the header so I could call it from the harness.

Reply:
Now I realize the ec_utils.h header is the place for these functions so everything in openssl/ec.h can remain openssl functions. I will change this accordingly.

Reply:
Updated!